src/sentry/web/frontend/pipeline_advancer.py

```diff
@@ -22,6 +23,11 @@ class PipelineAdvancerView(BaseView):
 
     def handle(self, request, provider_id):
         pipeline = None
+        if request.path == u'/extensions/github/setup/' and request.GET.get(
+                'setup_action') == 'install':
+            installation_id = request.GET.get('installation_id')
+            return self.redirect(reverse('integration-installation', args=[installation_id]))
```

Comment:
There's definitely a smell here with how hyper-specific this is. at minimum lets change it to check the provider_id instead of the path.

```py
FORWARD_INSTALL_FOR = ['github', 'github_enterprise']

# Github apps may be installed directly from GitHub, in which case 
# they will redirect here *without* being in the pipeline. If that happens 
# redirect to the integration install org picker.
if provider_id in FORWARD_INSTALL_FOR and request.GET.get('setup_action') == 'install':
    ...
```

We can probably have this live AFTER trying to lookup the pipeline too, since that's the priority.